### DISCUSSION TITLE
Admin route to reload repository rules

### DISCUSSION THREAD

**Discussion by EnzoAlbornoz** [ory/oathkeeper#659] [Ideas]
First of all, I liked a lot what you guys are doing here in this project. It fit on the right spot that my company was searching. 

Currently, we have a container running Oathkeeper (AWS ECS). Our microsservices requests Oathkeeper using the decisions API.

The problem is that when we are setting up a new service or adding new routes to them, we need to send the config to S3 bucket, update the config file in the file system and then restart Oathkeeper.

So, I think that a route to force a "reload" of the config routes via the Admin API would help a lot, because we can simply update S3 and then send a request (on deploy) to reload the repository routes.

**Comment by vinckr** [comment]
This sounds like a good addition to [Oathkeeper Next Gen](https://github.com/ory/oathkeeper/issues/441). 
Contributions & [feature requests](https://github.com/ory/oathkeeper/issues/new?assignees=&labels=feat&template=FEATURE-REQUEST.yml) always welcome 🤝

**Comment by jmls** [comment]
we need this :) Is it on the roadmap ?

**Reply by vinckr** [reply]
Check out the nextgen issue: https://github.com/ory/oathkeeper/issues/441 
Feel free to also [open a feature request](https://github.com/ory/oathkeeper/issues/new?assignees=&labels=feat&template=FEATURE-REQUEST.yml) with more details on how that feature should work etc.
